pkgs/misc/t-rec/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, imagemagick, ffmpeg, rustPlatform, fetchFromGitHub, makeWrapper }:
+
+let
+  binPath = lib.makeBinPath [
+    imagemagick
+    ffmpeg
+  ];
+in
+rustPlatform.buildRustPackage rec {
+  pname = "t-rec";
+  version = "0.5.2";
```

Comment:
"0.6.0" came out today, that brings up the question can I somehow contribute to keep it updated here? Or is there any auto PR-on-CI thing as it [exists for brew](https://github.com/marketplace/actions/homebrew-bump-formula)?

Reply:
Hi! We have the excellent @r-ryantm which creates PRs automatically updating a large number of packages in nixpkgs.

Thanks for this PR @Hoverbear, I'm very interested in using it. Would you like to push a version bump before we merge, or just go ahead and hit the big green button?

Reply:
@aanderse I've updated it! :) It's a great tool. It's been a bit of a challenge to create GIFs that I can embed within the twitter size limits, sadly.

@sassman Generally updates to this are "Pull nixpkgs, update the version number, try to build it with `nix build .#t-rec`, wait for a sha error, update it, repeat until the package builds and runs, commit with `t-rec: 0.5.2 -> 0.6.0` or similar"